pymake/makefile

```diff
@@ -43,108 +43,114 @@ ${SOURCEDIR14}
 
 # Define the Fortran compile flags
 FC = gfortran
-FFLAGS = -O2 -fbacktrace -ffpe-summary=overflow -ffpe-trap=overflow,zero,invalid 
+FFLAGS = -O2 -ffpe-summary=overflow -ffpe-trap=overflow,zero,invalid -fbacktrace --coverage
 
 # Define the C compile flags
 CC = gcc
-CFLAGS = -O3 -D_UF 
+CFLAGS = -O2 -D_UF 
```

Comment:
not sure if it matters anywhere, but here we go from -O3 to -O2

Reply:
Switched back to 3.  No reason for the change.